lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -215,6 +215,11 @@ e2function number entity:health()
 	return this:Health()
 end
 
+e2function number entity:maxHealth()
```

Comment:
Take this function and put it in player.lua, and then erase the function you created in player.lua. Then I can merge this PR. Right now you have two identical functions. This means one will overwrite the other.